frame/scheduler/src/lib.rs

```diff
@@ -269,175 +421,90 @@ decl_module! {
-					}
-
-					Some((order, index, *cumulative_weight, s))
-				})
-				.filter_map(|(order, index, cumulative_weight, mut s)| {
-					// We allow a scheduled call if any is true:
-					// - It's priority is `HARD_DEADLINE`
-					// - It does not push the weight past the limit.
-					// - It is the first item in the schedule
-					if s.priority <= schedule::HARD_DEADLINE || cumulative_weight <= limit || order == 0 {
-						let r = s.call.clone().dispatch(s.origin.clone().into());
-						let maybe_id = s.maybe_id.clone();
-						if let &Some((period, count)) = &s.maybe_periodic {
-							if count > 1 {
-								s.maybe_periodic = Some((period, count - 1));
-							} else {
-								s.maybe_periodic = None;
-							}
-							let next = now + period;
-							// If scheduled is named, place it's information in `Lookup`
-							if let Some(ref id) = s.maybe_id {
-								let next_index = Agenda::<T>::decode_len(now + period).unwrap_or(0);
-								Lookup::<T>::insert(id, (next, next_index as u32));
-							}
-							Agenda::<T>::append(next, Some(s));
-						} else {
-							if let Some(ref id) = s.maybe_id {
-								Lookup::<T>::remove(id);
-							}
-						}
-						Self::deposit_event(RawEvent::Dispatched(
-							(now, index),
-							maybe_id,
-							r.map(|_| ()).map_err(|e| e.error)
-						));
-						total_weight = cumulative_weight;
-						None
-					} else {
-						Some(Some(s))
-					}
-				})
-				.for_each(|unused| {
-					let next = now + One::one();
-					Agenda::<T>::append(next, unused);
-				});
-
-			total_weight
+			Ok(())
 		}
 	}
 }
 
-impl<T: Config> Module<T> {
+impl<T: Config> Pallet<T> {
 	/// Migrate storage format from V1 to V2.
 	/// Return true if migration is performed.
 	pub fn migrate_v1_to_t2() -> bool {
-		if StorageVersion::get() == Releases::V1 {
-			StorageVersion::put(Releases::V2);
+		if StorageVersion::<T>::get() == Releases::V1 {
+			StorageVersion::<T>::put(Releases::V2);
 
 			Agenda::<T>::translate::<
-				Vec<Option<ScheduledV1<<T as Config>::Call, T::BlockNumber>>>, _
-			>(|_, agenda| Some(
-				agenda
-					.into_iter()
-					.map(|schedule| schedule.map(|schedule| ScheduledV2 {
-						maybe_id: schedule.maybe_id,
-						priority: schedule.priority,
-						call: schedule.call,
-						maybe_periodic: schedule.maybe_periodic,
-						origin: system::RawOrigin::Root.into(),
-						_phantom: Default::default(),
-					}))
-					.collect::<Vec<_>>()
-			));
+ 				Vec<Option<ScheduledV1<<T as Config>::Call, T::BlockNumber>>>, _
+ 			>(|_, agenda| Some(
+ 				agenda
+ 					.into_iter()
+ 					.map(|schedule| schedule.map(|schedule| ScheduledV2 {
+ 						maybe_id: schedule.maybe_id,
+ 						priority: schedule.priority,
+ 						call: schedule.call,
+ 						maybe_periodic: schedule.maybe_periodic,
+ 						origin: system::RawOrigin::Root.into(),
+ 						_phantom: Default::default(),
+ 					}))
+ 					.collect::<Vec<_>>()
+ 			));
```

Comment:
nit: extra blank space at the beginning of every line

```suggestion
				Vec<Option<ScheduledV1<<T as Config>::Call, T::BlockNumber>>>, _
			>(|_, agenda| Some(
				agenda
					.into_iter()
					.map(|schedule| schedule.map(|schedule| ScheduledV2 {
						maybe_id: schedule.maybe_id,
						priority: schedule.priority,
						call: schedule.call,
						maybe_periodic: schedule.maybe_periodic,
						origin: system::RawOrigin::Root.into(),
						_phantom: Default::default(),
					}))
					.collect::<Vec<_>>()
			));
```

No idea why GitHub shows the diff as me erasing a tab...